frontend/lib/parse-capi/index.ts

```diff
@@ -7,19 +6,6 @@ import bigBullets from './big-bullets';
 import { pillarNames } from '../../pillars';
 import { getSharingUrls } from './sharing-urls';
 
-const headline = compose(
-    clean,
-    curly,
-);
-const standfirst = compose(
-    clean,
-    bigBullets,
-);
-const main = clean;
-const body = clean;
-
-const defaultArgs = { config: {}, contentFields: {} };
-
 // tslint:disable:prefer-array-literal
 const apply = (input: string, ...fns: Array<(_: string) => string>): string => {
```

Comment:
Is this function basically [`compose-function`](https://github.com/stoeffel/compose-function/blob/master/module/index.js)? Maybe this is a target for removal, since we already use `compose-function` elsewhere

Reply:
I do like this `apply` function but I think you're correct `compose` can achieve the same result, EG.

`apply(
    getNonEmptyString(data, 'config.page.headline'),
    clean,
    curly,
),`

Is equal to:

`const myComposition = compose(clean, curly);
myComposition(getNonEmptyString(data, 'config.page.headline'));`

I agree this  should be done as a separate PR though.